designs/penelope/src/front.mjs

```diff
@@ -2,6 +2,7 @@ import { pluginBundle } from '@freesewing/plugin-bundle'
 import { measurements, options, BuildMainShape } from './shape.mjs'
 
 function penelopeFront(part) {
+  // return( part )
```

Comment:
Was it intentional to commit this debug code?

Reply:
Nope, total oversight